packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -942,6 +942,27 @@ export default class Meetings extends WebexPlugin {
       if (res) {
         this.preferredWebexSite = MeetingsUtil.parseDefaultSiteFromMeetingPreferences(res);
       }
+
+      // fall back to getting the preferred site from the user information
+      if (!this.preferredWebexSite) {
+        // @ts-ignore
+        return this.webex.internal.user
+          .get()
+          .then((user) => {
+            const preferredWebexSite =
+              user?.userPreferences?.userPreferencesItems?.preferredWebExSite;
+            if (preferredWebexSite) {
```

Comment:
from the result point of view when `preferredWebexSite` is `undefined`, is the same as failing fetch. We should log this case as well.

You can and an else branch and throw an error so the catch will log both cases.

Reply:
I've now added this